catlas/filters.py

```diff
@@ -22,14 +21,17 @@ def bulk_filter(config, dask_df, sankey, initial_bulks):
         sankey: the sankey object with added info
     """
     bulk_filters = config["bulk_filters"]
-    sankey_idx = 2
-    sankey.info_dict["label"][1] = f"Bulks from db ({initial_bulks})"
+    columns = dask_df.columns
+
+    if sankey is not None:
+        sankey_idx = 2
+        sankey.info_dict["label"][1] = f"Bulks from db ({initial_bulks})"
 
     for name, val in bulk_filters.items():
         if (
             str(val) != "None"
         ):  # depending on how yaml is specified, val may either be "None" or NoneType
-            if name == "filter_by_bulk_ids":
+            if name == "filter_by_bulk_ids" and "bulk_id" in columns:
```

Comment:
Why is this necessary?

Reply:
this is how I am excluding filtering by bulk_id for parity because it would be too specific.